src/course-home/outline-tab/OutlineTab.jsx

```diff
@@ -173,7 +180,11 @@ const OutlineTab = ({ intl }) => {
                   <Section
                     key={sectionId}
                     courseId={courseId}
-                    defaultOpen={sections[sectionId].resumeBlock}
+                    defaultOpen={
+                      (selectedSectionId === null)
```

Comment:
```suggestion
                      (selectedSectionId === undefined)
```

The backport PR (https://github.com/open-craft/frontend-app-learning/pull/6) is not really a backport; usually, it's better to open an upstream PR for the review, and then cherry-pick the changes for the backport.